docs/source/features/fp8.rst

```diff
@@ -0,0 +1,40 @@
+FP8 usage
+=========
+
+Overview
+^^^^^^^^
+
+NVIDIA's H100 GPU introduced support for a new datatype, FP8 (8-bit floating point), enabling higher throughput of matrix multiplies and convolutions. NeMo uses NVIDIA's `TransformerEngine <https://github.com/NVIDIA/TransformerEngine>`_ (TE) in order to leverage speedups from FP8. The following table summarizes the FP8 related arguments that can be configured in NeMo (`example config setting <https://github.com/NVIDIA/NeMo/blob/2e1814c9f031ad2aeeebad44597365e97253d2c4/examples/nlp/language_modeling/conf/megatron_gpt_config.yaml/#L192-L200>`_). For a more detailed overview, refer to the TE `documentation <https://docs.nvidia.com/deeplearning/transformer-engine/user-guide/index.html>`_, specifically the FP8 `format <https://docs.nvidia.com/deeplearning/transformer-engine/user-guide/api/common.html#transformer_engine.common.recipe.Format>`_ and `recipe <https://docs.nvidia.com/deeplearning/transformer-engine/user-guide/api/common.html#transformer_engine.common.recipe.DelayedScaling>`_.
+
+.. list-table:: FP8 arguments
+   :widths: 25 25 50
+   :header-rows: 1
+
+   * - Argument
+     - Description
+   * - transformer_engine
+     - Boolean argument that needs to be set in order to enable TE and related functionality. All following arguments are ignored if this is not set to True.
```

Comment:
Change to "TE (True Enable) and related functionality can be enabled by setting the boolean argument to True. If this argument is not set to True, all subsequent arguments will be ignored."

Reply:
I have slight modified this. TE stands for TransformerEngine as opposed to True Enable.